src/zamp_collector/zcl_amp_c_bal_logs.clas.abap

```diff
@@ -0,0 +1,40 @@
+CLASS zcl_amp_c_bal_logs DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    INTERFACES zif_amp_collector.
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+ENDCLASS.
+
+
+
+CLASS zcl_amp_c_bal_logs IMPLEMENTATION.
+  METHOD zif_amp_collector~get_metrics.
+
+    metrics_current_run = zcl_amp_collector_utils=>initialize_metrics( metrics_last_run = metrics_last_run ).
+
+    SELECT
+    COUNT(*) AS count,
+    object AS object,
+    subobject AS subobject
+    FROM balhdr
+    INTO TABLE @DATA(bal_logs)
+    WHERE aldate = @sy-datum
```

Comment:
Could you use the date from the framework? 
That comes from a timestamp. If I do not mix up anything sy-datum has the OS date and the timestamp from the application server.

Reply:
yes i can
i also reviewed some older collectors and changed it to the frameworks date

Reply:
https://github.com/abap-observability-tools/abap-metrics-provider/pull/99/files#diff-969b6809c4e3cdd2dff3661b6e481c7d328ae0249f2021a22f06cbd38e6fc08fL22

https://github.com/abap-observability-tools/abap-metrics-provider/pull/99/files#diff-68ea295650e00988be4e40a30eae1055593b3835d28736d439fcc48976d4c63bL27

Reply:
well paid attention 😄